docs/product-user-guide/src/main/asciidoc/rules-attributes-ref.adoc

```diff
@@ -0,0 +1,60 @@
+
+*<@Michael (3 of 6): Could you again validate each of these attribute descriptions? It's like the one from the uploaded decision tables doc, but I've revised it pretty substantially for more clarity. I'll make the two tables more consistent once I get your go-ahead. I've also put in a UX request to just put in some help text in the UI for this, and they'll use what you/I/we come up with here. I have a few specific questions along the way that you'll see, but I've revised basically all of them, just as a heads-up.>*
+
+.Rule attributes
+[cols="30%,70%", options="header"]
+|===
+|Attribute
+|Value
+
+|`salience`
+|An integer defining the priority of the rule. Rules with a higher salience value are given higher priority when ordered in the activation queue.
+
+|`enabled`
+|A Boolean value. When the option is selected, the rule is enabled. When the option is not selected, the rule is disabled.
+
+|`date-effective`
+|A string containing a date and time definition. The rule can be activated only if the current date and time is after a `date-effective` attribute.
+
+|`date-expires`
+|A string containing a date and time definition. The rule cannot be activated if the current date and time is after the `date-expires` attribute.
+
+|`no-loop`
+|A Boolean value. When the option is selected, the rule cannot be reactivated (looped) if a consequence of the rule re-triggers a previously met condition. When the condition is not selected, the rule can be looped in these circumstances.
+
+|`agenda-group`
+|A string identifying an agenda group to which you want to assign the rule. Agenda groups allow you to partition the agenda to provide more execution control over groups of rules. Only rules in the agenda group that have acquired a focus are able to be activated. *<@Michael (4 of 6): Can you tell me what a focus is and what it's used for? It comes up again in `auto-focus`.>*
+
+|`activation-group`
+|A string identifying an activation (or XOR) group to which you want to assign the rule. In activation groups, only one rule can be activated. The first rule to fire will cancel all pending activations of all rules in the activation group.
+
+|`duration`
+|A long integer value defining the duration of time in minutes during which the rule can be activated. After the specified duration of time, the rule cannot be activated. *<@Michael (5 of 6): Is this accurate? And is it for certain in minutes? I see this as basically an expiration in minutes on the rule (similar to `date-expires`, but in minutes, not a date) and that it can fire within the duration period but not after. BUT the description in the GDT UI in Central says basically the opposite: "The duration dictates that the rule will fire after a specified duration, if it is still true." i.e., that it canNOT (right?) fire during the duration but can after, like a pause button. That's how I read it. Do you know which it is?>*
```

Comment:
The Community documentation states: "The duration dictates that the rule will fire **after** a specified duration, if it is still true" Since this is a core engine feature I'd have to simply agree with what those guys have put! @mariofusco @etirelli may know whether it's minutes (or seconds or hours.. it's not clear to be honest).

Reply:
Sorry @mfusco for the erroneous original mention).

Reply:
@sterobin The Community documentation does state (in it's "Timer/Calendars" section): "Rules now support both interval and cron based timers, which **replace the now deprecated duration attribute**".

Reply:
@manstis The duration is expressed in milliseconds

Reply:
But @manstis  and @mariofusco , the duration attribute is deprecated by timer and calendar now, and thus I've removed it from the doc. That right?

Reply:
@sterobin TBH I'd keep it in and mention that it's milliseconds... the UI still exposes the attribute; and it's deprecated not removed.

Reply:
Okay then.